ingestion/src/metadata/ingestion/source/database/postgres/query_parser.py

```diff
@@ -154,6 +154,7 @@ def process_table_query(self) -> Optional[Iterable[TableQuery]]:
                                 databaseName=self.get_database_name(row),
                                 serviceName=self.config.serviceName,
                                 databaseSchema=self.get_schema_name(row),
+                                duration=row.get("duration"),
```

Comment:
It might make sense to typecast this into `int` since the field is an `int` field

Reply:
+1